python/helper-image/launcher/launcher.go

```diff
@@ -252,6 +298,13 @@ func (pc *pythonContext) unwrapLauncher(_ context.Context) error {
 	return nil
 }
 
+func (pc *pythonContext) isPythonLauncher(ctx context.Context) error {
+	major, minor, err := determinePythonMajorMinor(ctx, pc.args[0], pc.env)
+	pc.major = major
```

Comment:
are all python version supported?  I see - https://github.com/GoogleContainerTools/container-debug-support/issues/67
would it make sense to error if 3.4 or below is found?

Reply:
I'm not sure it makes sense to hard-code it, especially since users can also put ptvsd or whatever into their requirements.txt.

Reply:
3.4 was released in 2014, so it's unlikely to occur in practice 🤞